### PR TITLE
nuspec

### PR DESCRIPTION
**Pull request by kristianGNI**
--Added nuspec
--Changed csproj to use the nuspec file
--Added nuget package as artifact

### CODE REVIEW

File: .circleci/config.yml
Comment (eikster-dk) [inline .circleci/config.yml]: Does this work? I think that the path should be from the "root" of the working directory and not from the src/dapper.... folder..

So I think it should look more like the following? 
```suggestion
          path: ./dist/Dapper.Azure.WebJobs.Extensions.SqlServer.${CIRCLE_TAG}.nupkg
```

File: src/Dapper.Azure.WebJobs.Extensions.SqlServer/Dapper.Azure.WebJobs.Extensions.SqlServer.csproj
Comment (eikster-dk) [inline src/Dapper.Azure.WebJobs.Extensions.SqlServer/Dapper.Azure.WebJobs.Extensions.SqlServer.csproj]: Would it make sense to have the nuspec file together with the library instead of <root>/nuget ?

**Review by eikster-dk — COMMENTED** [review]
I have a few comments before approval

File: src/Dapper.Azure.WebJobs.Extensions.SqlServer/Dapper.Azure.WebJobs.Extensions.SqlServer.csproj
Comment (kristianGNI) [reply src/Dapper.Azure.WebJobs.Extensions.SqlServer/Dapper.Azure.WebJobs.Extensions.SqlServer.csproj]: yeah, agree makes more sense

Comment (kristianGNI) [reply src/Dapper.Azure.WebJobs.Extensions.SqlServer/Dapper.Azure.WebJobs.Extensions.SqlServer.csproj]: i'll move it around

File: .circleci/config.yml
Comment (kristianGNI) [reply .circleci/config.yml]: both works

**Review by eikster-dk — DISMISSED** [review]

**Review by eikster-dk — DISMISSED** [review]

File: .circleci/config.yml
Comment (eikster-dk) [inline .circleci/config.yml]: Could you remove all the trailing whitespace ? :) It's really just generating noise :-) 

```suggestion
          command: dotnet pack -p:PackageVersion=${CIRCLE_TAG} -p:TargetFrameworks=netcoreapp2.2 --configuration Release -o ../../dist
```

**Review by eikster-dk — APPROVED** [review]

**Pull request merged**
